src/ring_theory/power_series/basic.lean

```diff
@@ -1188,6 +1188,15 @@ rescale (-1 : A)
 @[simp] lemma eval_neg_hom_X : eval_neg_hom (X : power_series A) = -X :=
 rescale_neg_one_X
 
+lemma sum_geo_seq (φ : power_series A) (n : ℕ) :
+((finset.range n).sum(λ k, φ^k) * (φ - 1)) = (φ^n - 1) :=
+begin
+  induction n with n h,
+  { simp only [finset.sum_empty, zero_mul, finset.range_zero, pow_zero, sub_self] },
+  simp only [finset.sum_range_succ, pow_succ', add_mul, h],
+  ring,
+end
```

Comment:
```suggestion
geom_sum_mul _ n
```
after `import algebra.geom_sum`, which makes me think this lemma does not belong here.

Reply:
Yes, makes sense. Thanks for taking the time!
Will close this pr.